client/src/components/RatingTile.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const RatingTile = ({ rating, id, handleInputChangeRating }) => {
+  const ratingFormatted = `Charm: ${rating.charm} | Cool: ${rating.cool} | Sharp: ${rating.sharp} | Tough: ${rating.tough} | Weird: ${rating.weird}`;
+  return (
+    <li id={id}>
+      <label htmlFor={`${rating}`} />
+      <input
+        type="radio"
+        name="rating"
+        value={JSON.stringify(rating)}
```

Comment:
hmmmmm I seeeeeeee. I don't love this as a way to store a set of data, but it does work. It would be better if every rating had some unique id, and all of the ratings were stored as an array of objects with your skill ratings. But I cant falter that his is way faster and easier to do. Maybe make a story to consider coming back to this